city_scrapers_core/commands/combinefeeds.py

```diff
@@ -159,6 +162,55 @@ def combine_azure(self):
             overwrite=True,
         )
 
+    def combine_gcs(self):
+        from google.cloud import storage
+
+        bucket_name = self.settings.get("GCS_BUCKET")
```

Comment:
I'm not familiar with Google Cloud Storage, but I'm assuming we can't parse the bucket out of the `FEED_URI` like S3 and Azure? That's fine with me as long as we document in `city-scrapers-template` that you'll need to set `GCS_BUCKET` to use GCS

Reply:
Great! Could we update the diff pipeline the same way?